### PR TITLE
LPS-193943 Error message for maximum number of characters for name does not informs which input is not attending the requirement

### PR DESCRIPTION
**Pull request by larissacribeiro**
**Related Issue**: [LPS-193943](https://liferay.atlassian.net/browse/LPS-193943)

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @murilostodolni 
I`m adding you because I believe it is related to this previous PR: https://github.com/liferay-objects/liferay-portal/pull/2491

**Comment by larissacribeiro** [comment]
Yes @marcelabc, you're right! Sorry, I forgot to link the previous one here.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/useObjectActionForm.ts [hunk 1]
```diff
@@ -112,7 +113,12 @@ export function useObjectActionForm({
 
 		if (Object.keys(errors).length) {
 			openToast({
-				message: REQUIRED_MSG,
+				message: sub(
+					Liferay.Language.get(
+						'only-x-characters-are-allowed-in-the-x-field'
+					),
+					['41', 'name']
+				),
```
Comment (murilostodolni) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/useObjectActionForm.ts:121]: If you add this message in this condition, when the user keep the ObjectAction name empty, the message on toast will be: `Only 41 characters are allowed in the name field`. I believe this message isn't idea for the empty field. 

So, I believe that you need a new conditional for length > 41, same like this:
```javascript
if (Object.keys(errors).length) {
	openToast({
		message: sub(
			Liferay.Language.get(
				'only-x-characters-are-allowed-in-the-x-field'
			),
			['41', 'name']
		),
		type: 'danger',
	});
}
```

What do you thing?

**Review by murilostodolni — CHANGES_REQUESTED** [review]
Hey @larissacribeiro

Thank you for your collaboration. I finished my review. Let me know if you have any question.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/useObjectActionForm.ts (hunk 1 above)
Comment (larissacribeiro) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/useObjectActionForm.ts:121]: Hi @murilostodolni, it makes sense, sorry for missing that. I will work on that now!

Comment (larissacribeiro) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/useObjectActionForm.ts:121]: @murilostodolni, I was about to commit my changes but I went to test the previous behavior in master to check if my changes weren't breaking anything and I saw that this message is already being brought from another file, probably the ones I've already altered, so there's no need to alter this one. I'll just remove my changes and update it now.

**Comment by larissacribeiro** [comment]
ci:test:sf

**Comment by larissacribeiro** [comment]
ci:test:relevant

**Comment by larissacribeiro** [comment]
ci:test:object

**Review by murilostodolni — APPROVED** [review]
LGTM!

@marcelabc could you double check, please?

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:forward:force

**Pull request closed without merging**